gt/main.c

```diff
@@ -597,6 +585,33 @@ gt_proc(void *arg)
 			for (i = num_tx_succ; i < num_tx; i++)
 				rte_pktmbuf_free(tx_bufs[i]);
 		}
+
+		if (ipv6_if_configured(&gt_conf->net->front) && acl.num > 0) {
+			int ret = rte_acl_classify(
+			    gt_conf->ipv6_acls[rte_lcore_to_socket_id(lcore)],
+			    acl.data, acl.res, acl.num, 1);
+			if (ret < 0) {
+				RTE_LOG(ERR, ACL, "%s: invalid arguments supplied to rte_acl_classify()\n",
+					__func__);
+				for (i = 0; i < acl.num; i++)
+					rte_pktmbuf_free(acl.mbufs[i]);
+				break;
```

Comment:
Yes, since if the ACL classification is wrong then it's a logical error, not a problem at runtime. Same with the GK block below.